cypress-io/cypress
# fix: report header stats overlapping issue with 4 digit test count

- Closes https://github.com/cypress-io/cypress/issues/21838
### User facing changelog

Fixed an bug where icon within stats of reporter header don't account for 4 digit test count. Fixes #21838



### Additional details
The number of tests (greater than 999) can't be seen clearly in header since the number used to be overlap with icon


### Steps to test

Start cypress app, select the target project and then start E2E testing
Start any specs (even example specs) on the header you can see the number of tests passed/failed

### How has the user experience changed?

Before
<img width="223" alt="171220139-2b4f959b-6336-453d-8320-fce29d0a175a" src="https://user-images.githubusercontent.com/51731962/183361450-d838a2d4-ed73-4b86-997c-dbf8c11c2884.png">

After
<img width="233" alt="Screen Shot 2022-08-08 at 12 54 46" src="https://user-images.githubusercontent.com/51731962/183361591-f4ea026a-5754-4eb7-90bd-6ff40f970e71.png">

### PR Tasks
